cumulusci/tasks/metadata_etl/remote_site_settings.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+from typing import List, Optional
+
+import pydantic
+from pydantic import BaseModel
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl.base import BaseMetadataSynthesisTask
+
+REQUIRED_FIELD_ERR = "You must specify a value for RemoteSiteSetting.{} on all entries."
+RSS_DIR_NAME = "remoteSiteSettings"
+RSS_FILE_EXTENSION = "remoteSite"
+RSS_XML = """<?xml version="1.0" encoding="UTF-8"?>
+<RemoteSiteSetting xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>{description}</description>
+    <disableProtocolSecurity>{disable_protocol_security}</disableProtocolSecurity>
+    <isActive>{is_active}</isActive>
+    <url>{url}</url>
+</RemoteSiteSetting>"""
+
+
+class RemoteSiteSetting(BaseModel):
+    """Schema for the RemoteSiteSetting object"""
+
+    full_name: str
+    url: str
+    is_active: bool
+    description: Optional[str]
+    disable_protocol_security: Optional[bool]
+
+
+class RSSOptions(BaseModel):
+    """Defines the options for this task"""
+
+    records: List[RemoteSiteSetting]
+
+
+class AddRemoteSiteSettings(BaseMetadataSynthesisTask):
+    """Task for adding Remote Site Setting records to an org"""
+
+    entity = "RemoteSiteSetting"
+    task_docs = """
+
+        Example Usage
+        -----------------------
+
+        .. code-block::  yaml
+
+            task: add_remote_site_settings
+            options:
+                records:
+                    - full_name: ExampleRemoteSiteSetting
+                      description: Descriptions can be included optionally
+                      url: https://test.salesforce.com
+                      is_active: True
+
+    """
+    task_options = {
+        "records": {
+            "description": "Array of RemoteSiteSetting records to insert. "
+            "Each RemoteSiteSetting requires the keys: 'full_name', 'is_active', and 'url'. "
+            "'description' is optional, and defaults to an empty string. "
+            "'disable_security_protocol' is optional, and defaults to False.",
+            "required": True,
+        },
+    }
+
+    def _get_package_xml_content(self, operation):
```

Comment:
This override is safe to remove.

Reply:
I thought so too. When I remove it from the class I get:
`TypeError: Can't instantiate abstract class OmniStudioDeployRemoteSiteSettings with abstract methods _get_package_xml_content`

Reply:
Oh. Huh. That's goofy. I'll take a quick look - that should be fixable.

Reply:
That's a mistake in the base class (my fault!) - could you move the override into `BaseMetadataSynthesisTask` and just have it return an empty string?